Plot_Data.py

```diff
@@ -0,0 +1,56 @@
+import Use_Data as data
+import matplotlib.pyplot as plt
+from pathlib import Path
+
+def plotdata(toPlot, filename, folder="Main_Sequences_txt_Files", 
+	title="", xtitle="", ytitle="", xlim=[0,0], ylim=[0,0]):
+	"""
+	plotdata takes in an array with the names of columns that are 
+	to be plotted, a filename with the data, and optionally a folder
+	which contains the files as well as many plotting options
+
+
+	Args:
+	    toPlot (np.array): array of names of values to be plotted
+	    filename (str): identifier for file
+	    folder (str): folder name without forward slash
+	"""
+
+	# Get data and header from file
+	arr, header = data.txt2array2D(filepath)
+
+	# Set up for the plotting loop
+	plotlen = len(toPlot)
+	headlen = len(header)
+	exists = False
+	label = -1
```

Comment:
```suggestion
	label = -1
	plt.figure(1, dpi=300)
```
This will just give the plots a bit more resolution.